recipes/app2.rb

```diff
@@ -124,3 +92,34 @@
   ]
   sensitive true
 end
+
+directory '/formsender'
+
+git '/var/lib/formsender' do
+  repository 'https://github.com/osuosl/formsender.git'
+  revision 'antoniagaete/RT_api'
+end
+
+docker_image 'formsender' do
+  tag 'latest'
+  source '/var/lib/formsender'
+  action :nothing
+end
+
+git '/var/lib/formsender' do
+  action :nothing
+  notifies :build, 'docker_image[formsender]', :immediately
+end
```

Comment:
Not sure why you're duplicating these `git` resources here. Just move the `notifies` to the `git` resource above and please remove this one.

Reply:
I assumed since the image pulls from git, I would have to sync git, pull the image, and then be able to notify git to build. If that's not the case, would all of the git resources go before I pull the image?

Reply:
That's not quite how it works. You need to define the `git` resource with the notify. Then you define the `docker_image` resource with the `nothing` action. Git will sync and then immediately notify the `docker_image`. We don't want to get into duplicating resources like you were doing here as it will not work correctly.